frame/support/src/weights.rs

```diff
@@ -620,7 +621,9 @@ impl<Call: Encode, Extra: Encode> GetDispatchInfo for sp_runtime::testing::TestX
 }
 
 /// The weight of database operations that the runtime can invoke.
-#[derive(Clone, Copy, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode)]
+#[derive(
+	Clone, Copy, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode, scale_info::TypeInfo,
```

Comment:
`TypeInfo` is already imported for this file, so we can just do this:

```suggestion
	Clone, Copy, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode, TypeInfo,
```